docs/source/components/nodes/color_camera.rst

```diff
@@ -0,0 +1,83 @@
+ColorCamera
+===========
+
+ColorCamera node is a source of :ref:`image frames <ImgFrame>`. You can control in at runtime with the :code:`InputControl` and :code:`InputConfig`.
+
+How to place it
+###############
+
+.. tabs::
+
+  .. code-tab:: py
+
+    pipeline = dai.Pipeline()
+    manip = pipeline.createColorCamera()
+
+  .. code-tab:: c++
+
+    dai::Pipeline pipeline;
+    auto imageManip = pipeline.create<dai::node::ColorCamera>();
+
+
+Inputs and Outputs
+##################
+
+.. code-block::
+
+  InitialControl ┌───────────────────┐   Still
+  ──────────────►│                   ├───────────►
+                 │                   │
+  InputConfig    │                   │   Preview
+  ──────────────►│    ColorCamera    ├───────────►
+                 │                   │
+  InputControl   │                   │   Video
+  ──────────────►│                   ├───────────►
+                 └───────────────────┘
+
+Message types
+#############
+
+- :code:`InitialControl` - :ref:`CameraControl`
+- :code:`InputConfig` - :ref:`ImageManipConfig`
+- :code:`InputControl` - :ref:`CameraControl`
+- :code:`Still` - :ref:`ImgFrame`
+- :code:`Preview` - :ref:`ImgFrame`
+- :code:`Video` - :ref:`ImgFrame`
+
+:code:`Preview` is RGB (or BGR planar/interleaved if configured) and is mostly suited for small size previews and to feed the image
+into :ref:`NeuralNetwork`. :code:`Video` and :code:`Still` are both NV12, so are suitable for bigger sizes. :code:`Still` image gets created when
+a capture event is sent to the ColorCamera, so it's like taking a photo.
```

Comment:
Can we use the reference of eg: `ColorCamera.inputConfig` - these usually have documentation associated with them:
```
/**
     * Input for ImageManipConfig message, which can modify crop paremeters in runtime
     *
     * Default queue is non-blocking with size 8
     */
```

Reply:
we could - with ` .. autoattribute:: depthai.ColorCamera.inputConfig` - but it wouldn't look good (it would take too much space)